src/components/Man/Man.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import { v1 as uuidv1 } from 'uuid';
+import { connect } from 'react-redux';
+import { Dispatch, Action } from 'redux';
+
+import { ManAllProps } from './models/man-types';
+import { savePeople } from '../../store/actions';
+import { PeopleCard } from '../../global-models';
+import { ApplicationState } from '../../store/reducers';
+
+import noimage from '../../img/noimage.png';
+
+const mapDispatchToProps = (dispatch: Dispatch<Action>) => ({
+  savePeople: (people: PeopleCard) => dispatch(savePeople(people))
+});
+
+const mapStateToProps = (state: ApplicationState) => ({
+  people: state.swPeople.people
+});
+
+const ConnectedMan = ({ man, index, people, savePeople }: ManAllProps) => {
+  const onDeleteCard = (e: any) => {
+    const clonedPeople = JSON.stringify(people);
+    const result = JSON.parse(clonedPeople);
```

Comment:
dont see types of `clonedPeople` and `result`. You do not need to describe everything that lives inside these variables if you're not going to invoke some of the inner values. in this case you can set something like `{}` or `object` or `[]`. Investigate it a little bit. If you can ignore types for common-named variables lite `mapStateToProps`, `mapDispatchToProps`, it doesn't mean that you can ignore it for all other custom-named variables.